### PR TITLE
Localhost run fix and added documentation

### PR DESCRIPTION
**Pull request by dgmouris**
This is one solution to the problem that will work, we could also make the settings allowed if DEBUG is true in settings but I feel like that might be a bit more cowboy.

Let me know what you think, any thoughts/reviews would be greatly appreciated.

Fixes: https://github.com/EdmontonPy/edmontonpy/issues/10

### CODE REVIEW

**Comment by KalobTaulien** [comment]
@dgmouris I like this PR. Mentioned not committing a local settings file, but also gives everyone a default local settings file to get the project up and running faster. 👍

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,5 @@
 # Django files
 edmontonpy/*.sqlite3
-edmontonpy/settings/local_settings.py
```
Comment (data-get) [inline .gitignore]: Lets keep this line, I don't want people accidentally adding their local_settings.py files to git.

Comment (dgmouris) [reply .gitignore]: I think we will be adding to the local_settings.py file in the future for some default settings to get people up and running in a reasonable amount of time.

Though I agree with you with accidental commits of local_settings.py so we'll keep it in there.

Thank you for the review 👍

**Comment by dgmouris** [comment]
Thanks @KalobTaulien I really appreciate the feedback :)

**Comment by data-get** [comment]
Due to the security concern around this, I won't be able to merge this until the web site has a valid certificate and is only serving traffic through HTTPS. Please keep this pull request open until that time. Thank you.

**Comment by dgmouris** [comment]
Thank you for the review @data-get :)

For reference for us (and those curious) the issue is here:
https://github.com/EdmontonPy/edmontonpy-ops/issues/2

**Comment by KalobTaulien** [comment]
@data-get What's the security concern exactly?

**Comment by data-get** [comment]
See: https://docs.djangoproject.com/en/2.1/topics/security/#host-headers-virtual-hosting

If `localhost` is set in production a user could craft a cross site scripting attack that would successfully be allowed to make requests to the website. If the admin interface was enabled that attack could be used to add/modify/remove content from the website without the user knowing.

**Comment by KalobTaulien** [comment]
@data-get  But this if for a local_setting.py file, is it not? Why would you be running local_settings.py in production?

**Comment by data-get** [comment]
`local_settings.py` is used to override settings. It is expected that deployment writes out its own `local_settings.py` file that being said the expectation is that the defaults should be secure.

**Comment by dgmouris** [comment]
I just changed this to add to the documentation and change the local_settings file to empty.

Hope everyone agrees with this!

**Pull request closed without merging**